src/lib/components/lines-and-dots/workflow-error.svelte

```diff
@@ -25,30 +22,12 @@
 
   let cause: WorkflowTaskFailedCause;
 
-  function getErrorCause(
-    error: WorkflowTaskFailedEvent,
-  ): WorkflowTaskFailedCause {
-    const {
-      workflowTaskFailedEventAttributes: { failure, cause },
-    } = error as WorkflowTaskFailedEvent & {
-      workflowTaskFailedEventAttributes: WorkflowTaskFailedEventAttributes;
-    };
-
-    if (
-      failure?.applicationFailureInfo?.type === 'workflowTaskHeartbeatError'
-    ) {
-      return 'WorkflowTaskHeartbeatError';
-    }
-
-    return toWorkflowTaskFailureReadable(cause);
-  }
-
   $: {
     cause = getErrorCause(error);
   }
 </script>
 
-{#if cause}
+{#if cause && cause !== 'ResetWorkflow'}
```

Comment:
I'm sure there will be more of these cases but not gonna overthink trying to abstract that way yet. This is fine.